icons/plasma.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Plasma</title><path d="M5.993 0a1.495 1.5 0 1 0 0 3.002 1.496 1.501 0 1 0 0-3.001zm10.505 0-3 3.002 4.5 4.498-4.5 4.5 3 3L21 10.5l3-3ZM2.242 9a2.242 2.25 0 1 0 0 4.5 2.242 2.25 0 1 0 0-4.5zm6.755 9a2.991 3.002 0 0 0-3.004 3 2.99 3.001 0 0 0 3.004 3 2.99 3 0 1 0 0-6z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KDE Plasma</title><path d="M5.993 0a1.495 1.5 0 1 0 0 3.002 1.496 1.501 0 1 0 0-3.001zm10.505 0-3 3.002 4.5 4.498-4.5 4.5 3 3L21 10.5l3-3ZM2.242 9a2.242 2.25 0 1 0 0 4.5 2.242 2.25 0 1 0 0-4.5zm6.755 9a2.991 3.002 0 0 0-3.004 3 2.99 3.001 0 0 0 3.004 3 2.99 3 0 1 0 0-6z"/></svg>
```

Reply:
Seeing issues with all 3 circles. Make sure not to over-optimize! 4 should be good here.
![CleanShot 2024-09-14 at 20 55 22](https://github.com/user-attachments/assets/6977b930-cb2f-4992-9a70-f887bdb90144)